### DISCUSSION TITLE
Build failed : [jee:deployear, doesnt exist in project "ejbca"]

### DISCUSSION THREAD

**Discussion by kharisma-iijgs** [Keyfactor/ejbca-ce#639] [Q&A]
Hi Experts,

I am new to EJBCA. And try to deploy ejbca after configuring wildfly 26.1.3. When running ant, the build is failed and there will be 1 error which said "jee:deployear, doesnt exist in project ejbca". I need some advice for this one, would you mind to help?

Thanks in advance.

**Comment by primetomas** [comment]
Ekat exact commands do you run, and what version of EJBCA?
Are you following the installation guide at https://docs.keyfactor.com/ejbca/latest/ejbca-installation. 

For the easiest deployment experience I can recommend the EJBCA container on DockerHub.
